src/activity_handlers.cpp

```diff
@@ -590,15 +590,15 @@ static void set_up_butchery( player_activity &act, player &u, butcher_type actio
     }
     bool b_rack_present = false;
     for( const tripoint &pt : g->m.points_in_radius( u.pos(), 2 ) ) {
-        if( g->m.has_flag_furn( flag_BUTCHER_EQ, pt ) ) {
+        if( g->m.has_flag_furn( flag_BUTCHER_EQ, pt ) || u.best_nearby_lifting_assist() >= 7 ) {
             b_rack_present = true;
         }
     }
     // workshop butchery (full) prequisites
     if( action == BUTCHER_FULL ) {
         const bool has_rope = u.has_amount( "rope_30", 1 ) || u.has_amount( "rope_makeshift_30", 1 ) ||
```

Comment:
Not specific to this PR, but this really should be a flag check instead.

Reply:
There is no appropriate flag that I know of, and making a flag seemed unnecessary.

Reply:
You should make a flag. Having a hardcoded list of item ids is bad.

Reply:
Alright, I will. I think cranes should probably have their own tag anyway to help program other functions for them.